src/module/postCommentMethod.js

```diff
@@ -0,0 +1,17 @@
+import getMovieDetails from './getMovieDetails';
+
+const postCommentMethod = () => {
+  const popup = document.querySelector('.popup');
+
```

Comment:
i would also recommend adding comments to your code in a way that is looks clear to your code reviewer